sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -170,22 +173,45 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
       {
         path: AppRoutes.RequestApplicationOfferingChange,
         name: InstitutionRoutesConst.REQUEST_CHANGE,
-        components: {
-          default: RequestApplicationChange,
-          sidebar: InstitutionHomeSideBar,
+        redirect: {
+          name: InstitutionRoutesConst.REQUEST_CHANGE_AVAILABLE_TO_CHANGE,
         },
         props: {
           default: (route: RouteLocationNormalized) => ({
             locationId: parseInt(route.params.locationId as string),
           }),
         },
+        components: {
+          default: RequestApplicationChange,
+          sidebar: InstitutionHomeSideBar,
+        },
         meta: {
           clientType: ClientIdType.Institution,
           institutionUserTypes: [
             InstitutionUserTypes.admin,
             InstitutionUserTypes.user,
           ],
         },
+        children: [
+          {
+            path: AppRoutes.RequestApplicationOfferingChangeAvailableToChange,
+            name: InstitutionRoutesConst.REQUEST_CHANGE_AVAILABLE_TO_CHANGE,
+            component: AvailableToChange,
+            props: true,
```

Comment:
I am explicitly passing the props to the child route and it works. So, I believe the below would work:
```
props: (route) => {
  return {
    locationId: parseInt(
      route.params.locationId as string,
    ),
  };
},
```